tests/cpp_unit_tests/test_main_model_sc.cpp

```diff
@@ -9,18 +9,19 @@
 namespace power_grid_model {
 TEST_CASE("Test main model - short circuit") {
     MainModel main_model{50.0};
+    ShortCircuitVoltageScaling const voltage_scaling = ShortCircuitVoltageScaling::maximum;
+    double const voltage_scaling_c = 1.1; // 1.1 for ShortCircuitVoltageScaling::maximum
```

Comment:
maybe you also want to add one or two more tests here

Reply:
done